packages/components/popover/src/popover.scss

```diff
@@ -1,31 +1,18 @@
 :host {
-  --_arrow-block-size: 6px;
-  --_arrow-inline-size: 16px;
-  --_background: var(--sl-color-popover-background);
-  --_border-color: var(--sl-color-popover-border);
-  --_border-radius: var(--sl-border-radius-popover-default);
-  --_border-width: 1px;
-  --_border: var(--_border-width) solid var(--_border-color);
-  --_box-shadow: var(--sl-elevation-shadow-md);
-  --_color: var(--sl-color-popover-foreground);
-  --_font: var(--sl-text-popover-text-text);
-  --_max-width: var(--sl-popover-max-inline-size, 80vw);
-  --_padding-block: var(--sl-space-popover-block);
-  --_padding-inline: var(--sl-space-popover-inline);
+  --_arrow-block-size: var(--sl-size-075);
+  --_arrow-inline-size: var(--sl-size-200);
 
-  background: var(--_background);
-  border: var(--_border);
-  border-radius: var(--_border-radius);
-  box-shadow: var(--_box-shadow);
+  background: var(--sl-elevation-surface-raised-default-idle);
+  border: var(--sl-size-borderWidth-subtle) solid var(--sl-color-border-plain);
+  border-radius: var(--sl-size-borderRadius-default);
+  box-shadow: var(--sl-elevation-shadow-overlay);
```

Comment:
Is it a proper box-shadow? In the previous version it was a little bit lighter I would say? :)